css/main.css

```diff
@@ -325,3 +325,130 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+#contact_form {
+  width: 375px;
+  margin: auto;
+  background: #fff;
+  height: 100%;
```

Comment:
- [ ] Kindly use flexbox to layout the elements as specified in the requirements